php/map.php

```diff
@@ -31,7 +31,7 @@
 $challenge = $_SESSION["challenge"] ?? null;
 if (!$uid || empty($uid)) {
     // If not logged in, default to demo mode and warn app that we're (no longer?) logged in
-    $uid = 1;
+    $uid = $_SESSION["opentrid"];
```

Comment:
What does a trip ID have to do with the user ID?

Reply:
I'm guessing I copied the wrong thing and wasn't paying any attention...